### DISCUSSION TITLE
Welcome to jeedom-wallbox Discussions!

### DISCUSSION THREAD

**Discussion by ioxFR** [ioxFR/jeedom-wallbox#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sebinfo77** [comment]
Bonjour,

J’ai supprimer mon username et password et depuis je ne peu plus ressaisi le username et password.
j’ai essayé de supprimer et réinstalle le plugin toujours le même problème.
pouvez-vous m’aider
cordialement
Seb
![Capture-wallbox](https://user-images.githubusercontent.com/117599449/200192190-c67d9994-930f-4e38-aa9d-58f3b14ac83f.JPG)

**Reply by ioxFR** [reply]
Bonjour @sebinfo77,

le problème est corrigé, je viens de publié une nouvelle version stable sur le jeedom market.

Bonne journée,
Valentin

**Comment by sebinfo77** [comment]
Bonjour,
je confirme c'est corrigé.
Merci
Sébastien
